partials/method.handlebars

```diff
@@ -90,20 +90,18 @@
       </div>
     {{/if}}
 
-    {{#if throws}}
-      <div class="return">
+    {{#throws}}
```

Comment:
Not exactly because it adds kind of context switching I guess. This is equivalent to:

```html
    {{#if throws}}
      <div class="throws">
        <h4>Throws:</h4>
        <div class="throws-description">
          {{#if throws.type}} <span class="type">{{#crossLink throws.type}}{{/crossLink}}</span>: {{/if}}
          {{#if throws.description}} {{{throws.description}}} {{/if}}
        </div>
      </div>
    {{/if}}
```

I can rewrite it to keep consistent with previous return section, it could be better.

To be honnest I was inspired by the default yuidocs theme [here](https://github.com/yui/yuidoc/blob/master/themes/default/partials/method.handlebars#L184) and I suppose that `#throws` support could come from a custom `blockHelperMissing` helper in yui [here](https://github.com/yui/yui3/blob/25264e3629b1c07fb779d203c4a25c0879ec862c/src/handlebars/js/handlebars-base.js#L69) but I could not verify it or go deeper